.github/workflows/github-actions.yml

```diff
@@ -1,5 +1,9 @@
 name: assignment-test
 on: [push]
+    tags-ignore:
+          - '*'
+    branches:
```

Comment:
@lnxblog do we need the branches setting here?  What happens if we don't include this?

Reply:
Another requirement was to prevent actions runner when creating new branches off of the master and pushing them. 
So this ensures only changes to master are tested

Reply:
Action run successful for test commit
https://github.com/cu-ecen-aeld/assignments-complete-private/actions/runs/3211400882
Did not run for branch creation
https://github.com/cu-ecen-aeld/assignments-complete-private/tree/yaml-test-run